doctrine/doctrine-bundle/2.0/config/packages/dev/doctrine.yaml

```
@@ -0,0 +1,4 @@
+doctrine:
+    dbal:
+        # backtrace queries in profiler (increases memory usage per request)
+        #profiling_collect_backtrace: '%kernel.debug%'
```

This can be uncommented, as it is activated/deactivated using `'%kernel-debug%'` parameter.

```suggestion
        # backtrace queries in profiler (increases memory usage per request)
        profiling_collect_backtrace: '%kernel.debug%'
```